bids-validator/bids_validator/rules/top_level_rules.json

```diff
@@ -133,5 +135,30 @@
     "tokens": {
       "@@@_other_top_files_ext_@@@": ["physio\\.json", "stim\\.json"]
     }
+  },
+
+  "microscopy_top": {
+    "regexp": "^[\\/\\\\](?:ses-[a-zA-Z0-9]+_)(?:_sample-[a-zA-Z0-9]+)(?:_chunk-[a-zA-Z0-9]+)?(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:@@@_microscopy_top_ext_@@@)$",
```

Comment:
Same comment as for session level, I think all entities should be optional at that level (ex: anat).